a3p-integration/proposals/g:gtm-fast-usdc/yarn.lock

```diff
@@ -1099,6 +1099,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@endo/eventual-send@npm:^1.3.1":
```

Comment:
Retorically, why no red in this diff?

Reply:
`npx yarn-deduplicate` does not work for these and newer `yarn` is supposed to dedupe automatically, so some other reason. I’m not inclined to dig into this, though.

Reply:
Yarn 2+ does have `yarn dedupe`

Reply:
And that does the trick. Reset the clock.